commons/build.gradle.kts

```diff
@@ -52,7 +52,7 @@ android {
     }
 
     configurations.all {
-        resolutionStrategy.cacheDynamicVersionsFor(0, TimeUnit.SECONDS)
+        resolutionStrategy.cacheChangingModulesFor(0, TimeUnit.SECONDS)
```

Comment:
which is the impact of this?

Reply:
Dynamic version is when we have a version specified with "+". Which we do not have. Changing version is when we have a dependency with the isChanging = true, of which we have 2, the sdk and the mobile ui. This way the dependency won't be cached and any new change will be downloaded. It was not working until now, that's why we had problems with some snapshot versions.